app/src/main/java/com/kickstarter/ui/viewholders/RewardsItemViewHolder.kt

```diff
@@ -11,7 +12,7 @@ class RewardsItemViewHolder(private val binding: RewardsItemViewBinding) : KSVie
         val rewardsItem = ObjectUtils.requireNonNull(data as RewardsItem?)
         val title = ksString.format(
             "rewards_info_item_quantity_title", rewardsItem.quantity(),
-            "quantity", ObjectUtils.toString(rewardsItem.quantity()),
+            "quantity", rewardsItem.quantity().numToString(),
```

Comment:
for kotlin I don't really think in makes sense, the language already provides `.toString()` in the primitive types